qfieldsync/gui/cloud_create_project_widget.py

```diff
@@ -127,6 +127,10 @@ def __init__(
             )
         )
 
+        self.projectOwnerRefreshButton.clicked.connect(
+            lambda: self.on_project_owner_refresh_button_click()
+        )
```

Comment:
this necessary? can't it just be `connect(self.on_project_owner_refresh_button_click)` ?

Reply:
Nope, you are missing the `self` in `on_project_owner_refresh_button_click` then. Theoretically you can define it as a class method, but it's ugly and in case you need the state in `on_project_owner_refresh_button_click`, you still need to have `self`.

Reply:
thanks for info :+1: